src/containers/SaveBarHandler/SaveBarButton/index.jsx

```diff
@@ -1,19 +1,23 @@
 import styles from '../style.postcss';
 
 import React from 'react';
+import classnames from 'classnames';
 import Button from 'components/Button';
 import PropTypes from 'prop-types';
+import testClass from 'domain/testClass';
 
-const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) =>
-  <Button autoWidth
+const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) => {
+  const className = classnames(styles.SaveBar_button, testClass(`${label}-btn`));
```

Comment:
`label` here is the text inside the button. That could be `Do Something`, with a space. The CSS class will be broken.